torch/nn/modules/upsampling.py

```diff
@@ -110,5 +115,21 @@ class UpsamplingBilinear2d(_UpsamplingBase):
 
     """
 
+    def __init__(self, size=None, scale_factor=None):
+        super(UpsamplingBilinear2d, self).__init__(size, scale_factor)
```

Comment:
As said in an earlier PR - can you move these checks to a helper function (prefixed with an underscore) in `functional.py` and call it from here?